central/debug/service/service.go

```diff
@@ -735,6 +750,17 @@ func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseW
 			return writeTelemetryData(zipWriter, telemetryData)
 		})
 	}
+	diagBundleTasks.Go(func(ctx context.Context) error {
+		data, err := s.getDeleRegConfigs(ctx)
+		if err != nil {
+			log.Error(err)
+		}
+		err = addJSONToZip(zipWriter, "delegated-scanning-config.json", data)
+		if err != nil {
+			log.Error(err)
+		}
+		return nil
+	})
```

Comment:
Could this be simplified:
```suggestion
	diagBundleTasks.Go(func(ctx context.Context) error {
		fetchAndAddJSONToZip(ctx, zipWriter, "delegated-scanning-config.json", s.getDeleRegConfigs)
		return nil
	})
```